Makefile

```diff
@@ -29,7 +31,7 @@ ifeq ($(HOST)$(TARGET),LinuxDarwin)
 $(error Can't build $(TARGET) on $(HOST).)
 endif
 ifneq ($(TARGET),Darwin)
-	MAKE := $(if $(filter $(HOST),Darwin),$(RUN),) $(MAKE)
+	MAKE := $(RUN) $(MAKE)
```

Comment:
This change is done to make sure we use Docker if the target is Linux even if we are already running in a posix environment. This is done so that github CI uses the docker container instead of expecting some other way to install the required dependencies.